testdata/baselines/reference/submoduleAccepted/compiler/ambientPropertyDeclarationInJs.errors.txt.diff

```diff
@@ -4,22 +4,24 @@
  /test.js(3,9): error TS2322: Type '{}' is not assignable to type 'string'.
 -/test.js(6,5): error TS8009: The 'declare' modifier can only be used in TypeScript files.
 -/test.js(6,19): error TS8010: Type annotations can only be used in TypeScript files.
++/test.js(4,6): error TS8009: The 'declare' modifier can only be used in TypeScript files.
++/test.js(6,18): error TS8010: Type annotations can only be used in TypeScript files.
  /test.js(9,19): error TS2339: Property 'foo' does not exist on type 'string'.
 
 
--==== /test.js (4 errors) ====
-+==== /test.js (2 errors) ====
-     class Foo {
-         constructor() {
-             this.prop = {};
-@@= skipped -12, +10 lines =@@
+@@= skipped -10, +10 lines =@@
+             ~~~~~~~~~
+ !!! error TS2322: Type '{}' is not assignable to type 'string'.
          }
++         
++    
      
          declare prop: string;
 -        ~~~~~~~
--!!! error TS8009: The 'declare' modifier can only be used in TypeScript files.
++    ~~~~~~~~~~~
```

Comment:
It looks like some of these ranges are wrong; the diagnostic positioning needs to skip the starting trivia, not just use the raw Pos of the nodes.